src/SourceLocator/SourceStubber/PhpStormStubsSourceStubber.php

```diff
@@ -81,21 +94,27 @@ public function __construct(Parser $phpParser)
         $this->nodeTraverser = new NodeTraverser();
         $this->nodeTraverser->addVisitor(new NameResolver());
         $this->nodeTraverser->addVisitor($this->cachingVisitor);
+
+        self::$classMap    = array_change_key_case(PhpStormStubsMap::CLASSES);
```

Comment:
this is done at every instantiation: since that will happen once (usually), we probably should use instance variables anyway, so we simply object lifecycle semantics

Reply:
Done.